src/components/space/ExportButton.js

```diff
@@ -46,11 +47,19 @@ class ExportButton extends Component {
     );
   }
 }
+
+const mapStateToProps = ({ authentication }) => ({
+  userId: authentication.getIn(['user', 'userId']),
```

Comment:
Here I see that inside `user` we are saving the ID as `userId`. I would've imagined that inside `user`, the user ID would be simply `id`.